source/config/__init__.py

```diff
@@ -727,13 +727,15 @@ def reset(self, factoryDefaults=False):
 		self._initBaseConf(factoryDefaults=factoryDefaults)
 		post_configReset.notify(factoryDefaults=factoryDefaults)
 
-	def createProfile(self, name):
+	def createProfile(self, name: str) -> None:

```

Comment:
There is no way for this method (and for that matter any other method modified in this file with the exception of `saveTriggers`) to be called in secure mode as they are invoked only from the profiles GUI and the appropriate controls are disabled in the secure mode. I think these messages are not needed here and should be reverted.

Reply:
fixed in c928794